packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -21,8 +21,12 @@ import { Resource } from '@opentelemetry/resources';
 /** The kind of metric. */
 export enum MetricKind {
   COUNTER,
+  UP_DOWN_COUNTER,
   VALUE_RECORDER,
-  OBSERVER,
+  OBSERVER, // deprecated
```

Comment:
I would add `@todo` note (optional: w/ issue#) here to remove it later.

Reply:
done